src/CRKSPH/SolidCRKSPHHydroBaseRZ.cc

```diff
@@ -161,11 +161,7 @@ initializeProblemStartup(DataBase<Dim<2> >& dataBase) {
     const unsigned n = mass[nodeListi]->numElements();
     for (unsigned i = 0; i != n; ++i) {
       const Scalar circi = 2.0*M_PI*abs(pos(nodeListi, i).y());
-#ifdef WIN32
```

Comment:
This (and the similar changes below for other RZ specializations) worry me.  We shouldn't be dividing by zero densities anywhere.  When is this happening? And why this protection for only the WIN32 platform?  If we really need to protect from div by zero we should use one of the safeInv functions (src/Utilities/safeInv.hh), but this shouldn't be happening at all.

Reply:
Ah yes, this was just for the mysterious extra elements we were getting on Fields with the Windows compiler. Since this is removing that extra hack, good to go.